airflow/example_dags/example_bash_operator.py

```diff
@@ -56,7 +56,7 @@
 for i in range(3):
     task = BashOperator(
         task_id='runme_' + str(i),
-        bash_command='echo "{{ task_instance_key_str }}" && sleep 1',
+        bash_command='echo "{{ task_instance_key_str }}" && sleep 60',
```

Comment:
I don't think you meant to commit this (and it'll probably slow down a number of tests as a result)